CMakeLists.txt

```diff
@@ -33,7 +33,7 @@
 ##
 ###############################################################################
 
-cmake_minimum_required (VERSION 3.9)
+cmake_minimum_required (VERSION 3.16)
```

Comment:
Why is 3.16 now required?  The version checks below that are for versions older than 3.16 can be made unconditional.

Reply:
For this which is at 3.10, but we had previously discussed moving to 3.16

https://cmake.org/cmake/help/latest/module/GoogleTest.html

Reply:
I guess I've forgotten but that's fine.  Please update the version checks.

Reply:
Done.